fluent-plugin-enhance-k8s-metadata/lib/fluent/plugin/filter_enhance_k8s_metadata.rb

```diff
@@ -33,7 +33,7 @@ class EnhanceK8sMetadataFilter < Fluent::Plugin::Filter
       # Need different clients to access different API groups/versions
       # https://github.com/abonas/kubeclient/issues/208
       config_param :core_api_versions, :array, default: ['v1']
-      config_param :api_groups, :array, default: ['apps/v1', 'extensions/v1beta1']
+      config_param :api_groups, :hash, default: {'apps':'v1', 'extensions':'v1beta1'}
```

Comment:
👍

Reply:
Wait what!?

it's a hash https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/60860d2b46ceb7772bf55981b61f19a33e78ed3c/fluent-plugin-enhance-k8s-metadata/lib/fluent/plugin/filter_enhance_k8s_metadata.rb#L36

o.0